python/lsst/dax/imgserv/templates/vosi_capabilities.xml

```diff
@@ -1,60 +1,64 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?xml-stylesheet type="text/css" href= "{{ url_for('api_image_soda.static', filename='style.css') }}"?>
 <vosi:capabilities
    xmlns:vosi="http://www.ivoa.net/xml/VOSICapabilities/v1.0"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-   xmlns:vod="http://www.ivoa.net/xml/VODataService/v1.1">
+   xmlns:vod="http://www.ivoa.net/xml/VODataService/v1.1"
+   xmlns:sia="http://www.ivoa.net/xml/SIA/v1.1">
```

Comment:
So you're adding in SIA?

Reply:
ImgServ or another closely VO service (like metaserv) will have to provide SIAv2 support; in fact, it's probably first thing users want to use, as part of prep before invoking SODA operations.  DM-14330 been opened for this work, that I will get to in near future.